### DISCUSSION TITLE
Raspberry Pi / Rock Pi Micro SD

### DISCUSSION THREAD

**Discussion by ivanbbctba** [Chia-Network/chia-blockchain#2407] [General]
Anyone knows the minimum size needed of the Micro SD for the Raspberry Pi / Rock Pi?

I ask this because I supose the Micro SD is where is created the temporary files to be moved to the HDD right? So it should be a 128gb Micro sd or bigger?

**Comment by cccat6** [comment]
No, your understanding is completely wrong.
Raspberry Pi is only for farming. Storing plots and plotting should be in the other machine.
Your Pi just connects the drive array or connects the NFS via network to access the plots on the drive array.
Pi is not strong enough to run a full node. It can only run harvester and you need to connect with a node.
Read more carefully and try to understand the whole structure before you working on actual mining.
https://github.com/Chia-Network/chia-blockchain/wiki/Network-Architecture
https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL
A lot of resources on the internet you can learn.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
